src/database.json

```diff
@@ -828,6 +834,12 @@
     "repo": "deno-watch",
     "desc": "A pure deno file watcher."
   },
+  "watch-v2": {
```

Comment:
CI is failing because deno.land/x doesn't allow dashes in the project name. Change this to watch_v2

If you click on the failing CI, you'll see this error message
```
      23 | test("database names with dashes are not allowed", () => {
      24 |   for (const key in DATABASE) {
    > 25 |     expect(key.includes("-")).toBeFalsy();
         |                               ^
      26 |   }
      27 | });
      28 | 
```

Reply:
Alright cheers, ill sort it now - I mistakenly thought that error was something unrelated!